From a78f709898f01159a0e264c56703e30870b184f5 Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Thu, 31 Aug 2017 18:51:05 -0500
Subject: [PATCH] Use SYSROOT when cross compiling.

Signed-off-by: John Jacques <john.jacques@intel.com>
---
 decoder/build/linux/makefile | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/decoder/build/linux/makefile b/decoder/build/linux/makefile
index d0c1efb..6eedb56b 100644
--- a/decoder/build/linux/makefile
+++ b/decoder/build/linux/makefile
@@ -84,6 +84,12 @@ MASTER_CPP_FLAGS += -g -O2 -DNDEBUG
 BUILD_VARIANT=rel
 endif
 
+ifdef SYSROOT
+MASTER_CC_FLAGS += --sysroot=$(SYSROOT)
+MASTER_CPP_FLAGS += --sysroot=$(SYSROOT)
+MASTER_LINKER_FLAGS += --sysroot=$(SYSROOT)
+endif
+
 # platform bit size variant
 ifeq ($(ARCH),x86)
    MFLAG:="-m32"
-- 
2.5.2

